### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by prashant5878-shukla**
Added missing icons as per #582 
Preview link: https://coder135792468.github.io/fossasia.org/

### CODE REVIEW

**Review by NishithP2004 — APPROVED** [review]
Great work with the crunch base icon but as a suggestion, could you please make it black and white to meet the other icons.

**Comment by prashant5878-shukla** [comment]
@NishithP2004 thanks I fixed it now the icon is matched to the other icons

**Review by NishithP2004 — APPROVED** [review]
Great Work 👍

**Comment by abishekvashok** [comment]
It doesn't look responsive in Mobile. Please take a look

File: index.html [hunk 1]
```diff
@@ -1651,7 +1651,14 @@ <h1>Friends and Partners</h1>
  								<li><a target="_self" href="https://gitlab.com/fossasia"><i class="fab fa-gitlab"></i></a></li>
 								<li><a target="_self" href="https://gitter.im/fossasia"><i class="fab fa-gitter"></i></a></li>
 								<li><a target="_self" href="https://de.wikipedia.org/wiki/FOSSASIA"><i class="fab fa-wikipedia-w"></i></a></li>
+           <li>
 
+<a target="_self" href="https://www.crunchbase.com/organization/fossasia">
+   <img style="width:20px;height:20px" src="http://gecad.com/wp-content/themes/gecad/assets/images/icon-crunchbase-black.svg"/>
```
Comment (abishekvashok) [inline index.html:1649]: Instead of using an svg, please use an icon provided by an imported icon pack (font awesome?)

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please intend the code as well

**Comment by NishithP2004** [comment]
That's the issue, Crunchbase isn't available in the Font awesome library.

**Comment by prashant5878-shukla** [comment]
@abishekvashok Sir crunch base icon not available in font awesome

**Comment by pranavkarthik10** [comment]
LGTM but if icon pack required, try a different one other than font awesome?

**Comment by prashant5878-shukla** [comment]
@pranavkarthik10 I can't any package of icons in which cruchbase icon is given so I add it using image

**Comment by prashant5878-shukla** [comment]
@abishekvashok  plz reply it going 2days without any reply

**Comment by prashant5878-shukla** [comment]
@mariobehling @hpdang or any other mentor plz look at this PR it's going for a 6 days without any reply

**Review by mariobehling — CHANGES_REQUESTED** [review]
Well, instead of hosting the file externally, please add it to the image folder.

**Comment by prashant5878-shukla** [comment]
@mariobehling please take a look and merge it

**Review by mariobehling — CHANGES_REQUESTED** [review]
This was tried already by several people. The approach breaks the display of Github icons in other areas.

**Comment by prashant5878-shukla** [comment]
@mariobehling this is a task of gci .so Please merge it. 
And please tell me what is the problems in the code ?

**Comment by pranavkarthik10** [comment]
@coder135792468 This requires an update of font awesome, which requires a change of all the github icons website wide. Without changing each to fab, they will be rendered incorrectly.

**Comment by prashant5878-shukla** [comment]
@pranavkarthik10 bro crunch base icon is not in any font awesome library . And other icons are added correctly .
You can see them by preview links

**Comment by mariobehling** [comment]
Changes result in not showing the GitHub icon on team page anymore: https://coder135792468.github.io/fossasia.org/team/

The code for the existing icons should not be changed.

**Comment by prashant5878-shukla** [comment]
@mariobehling sir the changes are showing
Screenshot_2020-01-09-17-00-15-35

Look at it please

**Comment by prashant5878-shukla** [comment]
@mariobehling @hpdang plz review and merge it

**Comment by prashant5878-shukla** [comment]
@mariobehling @hpdang anyone plz merge it

**Comment by prashant5878-shukla** [comment]
@abishekvashok @mariobehling @hpdang @cweitat anyone please see it .
No one is replying .
Any mentor plz review it 🙏🙏🙏🙏🙏🙏🙏🙏

**Pull request closed without merging**